magento/magento2
# Convert MSRP currency of configurable product options

### Description

The MSRP of configurable products is updated in Javascript when an option is chosen. Javascript uses an object with prices of the options. The prices in this object are converted to the chosen currency. MSRP prices however were not converted, leading to the price being displayed in a wrong currency.

### Manual testing scenarios

1. Enable MAP (Configuration > Sales > Sales > Minimum Advertised Price > Enable MAP).
2. Enable multiple currencies (Configuration > General > Currency Setup > Allowed Currencies)
3. Create a configurable product with options that have an MSRP
4. Go to the frontend and choose a currency that is not the base currency
5. View the product on the frontend a choose an option.

Expected result: The MSRP is converted to the chosen currency
Actual result: The MSRP is not converted but it is labeled as the chosen currency

![Screenshot from 2020-03-26 11-13-38](https://user-images.githubusercontent.com/45412158/77610665-4eed3980-6f5e-11ea-8dbb-bb44991af8ba.png)

Note that the MSRP is configured in euro but displayed in dollars with the euro amount.

After applying the change the MSRP is converted:

![Screenshot from 2020-03-26 12-52-14](https://user-images.githubusercontent.com/45412158/77611623-c58b3680-6f60-11ea-9c65-b101913ff225.png)

Rates in table directory_currency_rate:

![Screenshot from 2020-03-26 12-57-40](https://user-images.githubusercontent.com/45412158/77611862-6bd73c00-6f61-11ea-9bb0-7381ac332fc2.png)


### Resolved issues:
1. [x] resolves magento/magento2#29603: Convert MSRP currency of configurable product options
